### PR TITLE
optimize:optimize time query conditions in the console

### LINKED ISSUE #6272: seata 控制台查询事务列表接口 时间筛选无效
无时间筛选
image
gmtcreate 为 1705557870000 

选择的了时间筛选
image

查询不出数据

版本 seata1.7.1

### PR DESCRIPTION
**Pull request by xingfudeshi**
- [x] I have registered the PR [changes](../changes).

### Ⅱ. Does this pull request fix one issue?

fixes #6272 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
I've already tested file mode and database mode locally
image

### CODE REVIEW

**Comment by slievrly** [comment]
some UT failed.

File: server/src/main/java/io/seata/server/console/impl/db/GlobalLockDBServiceImpl.java [hunk 1]
```diff
@@ -134,12 +137,12 @@ private String getWhereConditionByParam(GlobalLockParam param, List<Object> sqlP
             sqlParamList.add(param.getBranchId());
         }
         if (param.getTimeStart() != null) {
-            whereConditionBuilder.append(" and gmt_create >= ? ");
-            sqlParamList.add(param.getTimeStart());
+            whereConditionBuilder.append(" and FROM_UNIXTIME(begin_time/1000,'%Y-%m-%d %H:%i:%s') >= ? ");
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/console/impl/db/GlobalLockDBServiceImpl.java]: This function only works with MySQL.

Comment (xingfudeshi) [reply server/src/main/java/io/seata/server/console/impl/db/GlobalLockDBServiceImpl.java]: ok.will fix it.

File: common/src/main/java/io/seata/common/util/PageUtil.java [hunk 2]
```diff
@@ -169,4 +169,42 @@ public static void setObject(PreparedStatement ps, List<Object> sqlParamList) th
...
     }
+
+    /**
+     * get sql for time start
+     * @param dbType
+     * @param timeColumnName
+     * @return java.lang.String
+     */
+    public static String getTimeStartSql(String dbType, String timeColumnName) {
+        switch (dbType) {
```
Comment (slievrly) [inline common/src/main/java/io/seata/common/util/PageUtil.java]: to lower case.

File: common/src/main/java/io/seata/common/util/PageUtil.java [hunk 3]
```diff
@@ -169,4 +169,42 @@ public static void setObject(PreparedStatement ps, List<Object> sqlParamList) th
...
+    }
+
+    /**
+     * get sql for time end
+     * @param dbType
+     * @param timeColumnName
+     * @return java.lang.String
+     */
+    public static String getTimeEndSql(String dbType, String timeColumnName) {
+        switch (dbType) {
```
Comment (slievrly) [inline common/src/main/java/io/seata/common/util/PageUtil.java]: same as above.

File: common/src/main/java/io/seata/common/util/PageUtil.java (hunk 2 above)
Comment (xingfudeshi) [reply common/src/main/java/io/seata/common/util/PageUtil.java]: Done.

File: common/src/main/java/io/seata/common/util/PageUtil.java (hunk 3 above)
Comment (xingfudeshi) [reply common/src/main/java/io/seata/common/util/PageUtil.java]: Done.

**Review by slievrly — APPROVED** [review]
LGTM
In terms of function, this actually belongs to bugfix.

**Comment by xingfudeshi** [comment]
> LGTM In terms of function, this actually belongs to bugfix.

ok

**Pull request merged**